VMBackup/main/snapshotter.py

```diff
@@ -304,7 +304,7 @@ def snapshotall(self, paras, freezer):
         else:
             snapshot_result, snapshot_info_array, all_failed, exceptOccurred, is_inconsistent, thaw_done, unable_to_sleep =  self.snapshotall_parallel(paras, freezer, thaw_done)
             if exceptOccurred and thaw_done == False:
-                snapshot_result, snapshot_info_array, all_failed, exceptOccurred, is_inconsistent, unable_to_sleep =  self.snapshotall_seq(paras, freezer)
+                snapshot_result, snapshot_info_array, all_failed, exceptOccurred, is_inconsistent, unable_to_sleep =  self.snapshotall_seq(paras, freezer, thaw_done)
```

Comment:
thaw_done return value is not being taken from output values

snapshot_result, snapshot_info_array, all_failed, exceptOccurred, is_inconsistent, **thaw_done**, unable_to_sleep =  self.snapshotall_seq(paras, freezer, thaw_done)